src/commands/config/levelRole/index.ts

```diff
@@ -43,7 +43,7 @@ const command: Command = {
     }
 
     const level = +levelArg
-    if (isNaN(level) || level === 0)
+    if (isNaN(level) || level <= 0)
```

Comment:
```suggestion
    if (Number.isNaN(level) || level <= 0 || level >= Infinity)
```
có 2 chỗ: `isNaN()` và chưa handle case Infinity

tham khảo link này nhe https://www.designcise.com/web/tutorial/what-is-the-difference-between-isnan-and-number-isnan-in-javascript

tóm tắt: `isNaN()` _**convert**_ và check, còn `Number.isNaN()` chỉ check thôi nên nó ổn định hơn, thằng kia thì nó tự động convert qua nên nhiều khi convert ra nhiều output không đoán được